### PR TITLE
[ACM-5456]: Updates to AWS docs

### PR DESCRIPTION
**Pull request by lahinson**
Related Jira: https://issues.redhat.com/browse/ACM-5456

This PR applies fixes to issues that were noted in Red Hat internal user testing.

### CODE REVIEW

**Comment by lahinson** [comment]
@bryan-cox PTAL when you can. Thanks!

**Review by bryan-cox — APPROVED** [review]
/lgtm

File: clusters/hosted_control_planes/hosted_deploy_cluster_aws.adoc [hunk 1]
```diff
@@ -51,3 +52,10 @@ hcp create cluster aws \
 ----
 oc get hostedclusters -n <hypershift-hosting-service-cluster>
 ----
+
+. You can check your node pools by running the following command:
+
++
```
Comment (dockerymick) [inline clusters/hosted_control_planes/hosted_deploy_cluster_aws.adoc:61]:
```suggestion
+
[source,bash]
```

File: clusters/hosted_control_planes/hypershift_cluster_destroy_aws.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ where `_cluster_name_` is the name of your cluster.
 
 +
 ----
```
Comment (dockerymick) [inline clusters/hosted_control_planes/hypershift_cluster_destroy_aws.adoc:21]:
```suggestion
[source,bash]
----
```

**Review by dockerymick — APPROVED** [review]
recommend adding the `source` syntax. Otherwise, this looks good!

**Pull request merged**